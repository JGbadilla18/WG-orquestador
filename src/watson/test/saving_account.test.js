const testWatson = require('./testWatson');

watsonText = 'cuenta monetaria.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});

watsonText2 = 'cuenta monetaria super cuenta.'

it(watsonText2, () => {
  expect.assertions(1);
  return testWatson(watsonText2).then(data => expect(data).toMatchSnapshot());
});

watsonText3 = 'cuenta monetaria corriente.'

it(watsonText3, () => {
  expect.assertions(1);
  return testWatson(watsonText3).then(data => expect(data).toMatchSnapshot());
});

watsonText4 = 'cuenta monetaria oro.'

it(watsonText4, () => {
  expect.assertions(1);
  return testWatson(watsonText4).then(data => expect(data).toMatchSnapshot());
});

watsonText5 = 'cuenta monetaria oro dolares.'

it(watsonText5, () => {
  expect.assertions(1);
  return testWatson(watsonText5).then(data => expect(data).toMatchSnapshot());
});