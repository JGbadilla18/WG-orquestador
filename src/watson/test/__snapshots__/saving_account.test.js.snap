// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cuenta monetaria corriente. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para la apertura de una Cuenta Monetaria debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:

  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  NIT
  *  Depositar el monto mínimo de apertura según cuenta",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-corriente-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Corriente.jpg",
          "subtitle": "Obtienes disponibilidad de tus fondos a través de una chequera o tu Bi Cheque.",
          "title": "Cuenta Corriente",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/super-cuenta-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Super-Cuenta-Monetaria.jpg",
          "subtitle": "Dispones de tus fondos a través de chequera o Bi Cheque y acumulas Bi Puntos.",
          "title": "Super Cuenta",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-quetzales-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Quetzales-1.jpg",
          "subtitle": "A través de tu chequera o tu Bi Cheque tienes disponibilidad de tus fondos.",
          "title": "Cuenta Oro Quetzales",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-dolares-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Dolares-1.jpg",
          "subtitle": "Única cuenta en dólares, en donde dispones de tus fondos de forma inmediata.",
          "title": "Cuenta Oro Dólares",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;

exports[`cuenta monetaria oro dolares. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para la apertura de una Cuenta Monetaria debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:

  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  NIT
  *  Depositar el monto mínimo de apertura según cuenta",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-dolares-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Dolares-1.jpg",
          "subtitle": "Única cuenta en dólares, en donde dispones de tus fondos de forma inmediata.",
          "title": "Cuenta Oro Dólares",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-quetzales-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Quetzales-1.jpg",
          "subtitle": "A través de tu chequera o tu Bi Cheque tienes disponibilidad de tus fondos.",
          "title": "Cuenta Oro Quetzales",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-corriente-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Corriente.jpg",
          "subtitle": "Obtienes disponibilidad de tus fondos a través de una chequera o tu Bi Cheque.",
          "title": "Cuenta Corriente",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/super-cuenta-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Super-Cuenta-Monetaria.jpg",
          "subtitle": "Dispones de tus fondos a través de chequera o Bi Cheque y acumulas Bi Puntos.",
          "title": "Super Cuenta",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;

exports[`cuenta monetaria oro. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para la apertura de una Cuenta Monetaria debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:

  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  NIT
  *  Depositar el monto mínimo de apertura según cuenta",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-quetzales-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Quetzales-1.jpg",
          "subtitle": "A través de tu chequera o tu Bi Cheque tienes disponibilidad de tus fondos.",
          "title": "Cuenta Oro Quetzales",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-corriente-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Corriente.jpg",
          "subtitle": "Obtienes disponibilidad de tus fondos a través de una chequera o tu Bi Cheque.",
          "title": "Cuenta Corriente",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/super-cuenta-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Super-Cuenta-Monetaria.jpg",
          "subtitle": "Dispones de tus fondos a través de chequera o Bi Cheque y acumulas Bi Puntos.",
          "title": "Super Cuenta",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-dolares-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Dolares-1.jpg",
          "subtitle": "Única cuenta en dólares, en donde dispones de tus fondos de forma inmediata.",
          "title": "Cuenta Oro Dólares",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;

exports[`cuenta monetaria super cuenta. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para la apertura de una Cuenta Monetaria debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:

  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  NIT
  *  Depositar el monto mínimo de apertura según cuenta",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/super-cuenta-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Super-Cuenta-Monetaria.jpg",
          "subtitle": "Dispones de tus fondos a través de chequera o Bi Cheque y acumulas Bi Puntos.",
          "title": "Super Cuenta",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-corriente-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Corriente.jpg",
          "subtitle": "Obtienes disponibilidad de tus fondos a través de una chequera o tu Bi Cheque.",
          "title": "Cuenta Corriente",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-quetzales-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Quetzales-1.jpg",
          "subtitle": "A través de tu chequera o tu Bi Cheque tienes disponibilidad de tus fondos.",
          "title": "Cuenta Oro Quetzales",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-dolares-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Dolares-1.jpg",
          "subtitle": "Única cuenta en dólares, en donde dispones de tus fondos de forma inmediata.",
          "title": "Cuenta Oro Dólares",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;

exports[`cuenta monetaria. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Para la apertura de una Cuenta Monetaria debes acercarte a Servicio al Cliente en cualquiera de nuestras Agencias Bi y presentar la siguiente papelería:

  *  DPI
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses
  *  NIT
  *  Depositar el monto mínimo de apertura según cuenta",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/super-cuenta-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Super-Cuenta-Monetaria.jpg",
          "subtitle": "Dispones de tus fondos a través de chequera o Bi Cheque y acumulas Bi Puntos.",
          "title": "Super Cuenta",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-corriente-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Corriente.jpg",
          "subtitle": "Obtienes disponibilidad de tus fondos a través de una chequera o tu Bi Cheque.",
          "title": "Cuenta Corriente",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-quetzales-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Quetzales-1.jpg",
          "subtitle": "A través de tu chequera o tu Bi Cheque tienes disponibilidad de tus fondos.",
          "title": "Cuenta Oro Quetzales",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/cuenta-oro-dolares-monetaria",
            },
            Object {
              "title": "Solicítala",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-cuentas-monetarias?utm_campaign=SolicitudEnLinea&utm_medium=PortalBi&utm_source=SitioSolicitudes&utm_content=CuentaMonetaria",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Cuenta-Monetaria-Dolares-1.jpg",
          "subtitle": "Única cuenta en dólares, en donde dispones de tus fondos de forma inmediata.",
          "title": "Cuenta Oro Dólares",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
